source/_data/glossary.yml

```
@@ -292,6 +292,14 @@
 
   link: /docs/scripts/service-calls/
 
+- term: State
+  definition: |-
+    State holds the information of interest of an entity. E.g. if a light is on or off, the current temperature, or the amount of energy used. The data type of state is `String`. Entities store 2 timestamps related to state: `last_updated` and `last_changed`. Each entity has exactly one state and state only holds one value at a time. However, entities can store attributes related to that state. E.g. the state of a light is _on_, and the related attributes are `brightness: 102`, `friendly name: Coffee light`.
```

- Small wording corrections.
- Added a small note/clarification on what a string is
- Removed the more concrete examples of state attributes in the end to a more human-readable text. This was mostly triggered by the `friendly_name` (as it isn't really an actual enlightenment on the state value).

```suggestion
    The state holds the information of interest of an entity. E.g. if a light is on or off, the current temperature, or the amount of energy used. The data type of state is `string` (a textual value). Entities store 2 timestamps related to the state: `last_updated` and `last_changed`. Each entity has exactly one state and the state only holds one value at a time. However, entities can store attributes related to that state. E.g. the state of a light is _on_, and the related state attributes could be its current brightness and color values.
```

Let me know what you think.

---

Love the improvements. :rocket: 
I think I need to remind myself from time to time that it is _user_ documentation we are doing here ;-)